types/ember__test-helpers/setup-context.d.ts

```diff
@@ -1,7 +1,8 @@
 import Resolver from 'ember-resolver';
+import { TestContext } from './index';
 
 export default function setupContext<C extends object>(context: C, options?: { resolver?: Resolver | undefined }): Promise<C>;
-export function getContext(): object;
+export function getContext(): TestContext;
```

Comment:
You can see the reason that this isn't actually valid on the line immediately below this. While it is *very annoying* that the test context might not actually be a `TestContext`, the reality is that `setContext` could have been called with something *completely* unrelated to `TestContext`, and there is, in the present API design, *no way to relate the two types*. 😩 

Example of a broken flow:

```ts
module('this stinks', function (hooks) {
  hooks.beforeEach(function () {
    setContext({ lol: '🤨' });
  });

  test('uh oh', function (assert) {
    // here, `context` is `{ lol: '🤨' }`, *not* `TestContext`
    // 😭 😭 😭 😭 😭
    const context = getContext();
  });
});